core/theme/dark.js

```diff
@@ -116,39 +26,3 @@ Blockly.Themes.Dark.setComponentStyle('flyoutForegroundColour', '#ccc');
 Blockly.Themes.Dark.setComponentStyle('flyoutOpacity', 1);
 Blockly.Themes.Dark.setComponentStyle('scrollbarColour', '#797979');
 Blockly.Themes.Dark.setComponentStyle('scrollbarOpacity', 0.4);
```

Comment:
Could we add these to the object we pass into the define theme so we have an example of best way to use it?

Reply:
Great idea!